roles/php/tasks/install_php.yml

```diff
@@ -66,10 +66,3 @@
   when: item.when|default(true)
   become: true
   with_items: "{{ php_custom_config }}"
-
-- name: "{{ php_current_version }} - Add ~/.load_php file to add PHP to $PATH"
```

Comment:
sphp with current php version is call at the end of the php role so this task is useless